core/src/components/menu-controller/menu-controller.ts

```diff
@@ -63,35 +75,44 @@ export class MenuController implements MenuControllerI {
   }
 
   /**
-   * Used to enable or disable a menu. For example, there could be multiple
-   * left menus, but only one of them should be able to be opened at the same
-   * time. If there are multiple menus on the same side, then enabling one menu
-   * will also automatically disable all the others that are on the same side.
+   * Enable or disable a menu. This is useful when there are multiple
+   * menus on the same side and only one of them should be allowed to
+   * open. Enabling a menu will automatically disable all other menus
```

Comment:
What does it mean for a menu to be disabled? Does it just mean it won't respond to swipe events or subsequent calls to `open`?